README.md

```diff
@@ -1,48 +1,94 @@
+* [Tensorflow: Softmax / Fully-connected layer](flowers) - How to train a fully connected model with softmax using an image dataset of flowers to recognize the type of a flower from its image.
+
+### Hyperparameter Tuning (HP Tuning)
+
+Code Guide:
+* [Keras: Sequential / Dense](census/keras) - [Keras](https://keras.io/) - How to train a Keras sequential and Dense model using a text based dataset, Census, to predict a person’s income level using a single node model.
+* [Tensorflow Pre-made Estimator: Deep Neural Network Linear Combined Classifier](census/estimator) -How to train a DNN using Tensorflow’s Pre-made Estimators using a text based dataset, Census, to predict a person’s income level. [TensorFlow Pre-made Estimator](https://www.tensorflow.org/programmers_guide/estimators#pre-made_estimators), an estimator is “a high-level TensorFlow API that greatly simplifies machine learning programming.”
+* [Tensorflow Custom Estimator: Deep Neural Network](census/customestimator) - How to train a DNN using Tensorflow’s Custom Estimators using a text based dataset, Census, to predict a person’s income level. [TensorFlow Custom Estimator](https://www.tensorflow.org/programmers_guide/estimators#custom_estimators), which is when you write your own model function. 
+* [Tensorflow: Deep Neural Network](census/tensorflowcore) - How to train a DNN using TensorFlow’s low level APIs to create your DNN model on a single node using a text based dataset, Census, to predict a person’s income level.
+* [Tensorflow: Matrix Factorization / Deep Neural Network with Softmax](movielens) - How to train a Matrix Factorization and DNN with Softmax using a text based dataset, MovieLens 20M, to make movie recommendations.
+
+# Cloud TPU
+
+Please see the [Cloud TPU guide](CLOUD_TPU_README.md) for how to use Cloud TPU. 
+
+
+# Resources
+
+* [TensorFlow Estimator Trainer Package Template](cloudml-template) - When training a Tensorflow model, you have to create a trainer package, here we have a template that simplifies creating a trainer package for Cloud ML Engine.
+* Tensorflow to TF-Lite
 
 ## Google Samples
 
 * [Genomics Ancestry Inference](https://github.com/googlegenomics/cloudml-examples) - Genomics ancestry inference using 1000 Genomes dataset
 
-- - -
+# What do you want to see?
+
+If you came looking for a sample we don’t have, please file an issue on this repository and add the label `SAMPLE_REQUEST`. Please provide as much detail as possible about the ML Engine sample you were looking for, what framework (Tensorflow, Keras, scikit-learn, XGBoost, ...), the type of model, and what kind of dataset you were hoping to use! 
+
+Sample request template:
+```
+Workflow (Training, Serving, Complete Guide):
+ML Engine specfic features (CPU, GPU, HP Tuning, TPU):
+Framework (Tensorflow, Keras, scikit-learn, XGBoost, ...):
+Model:
+Dataset:
+Detailed information:
+```
+
+Jump below if you want to contribute and add that missing sample.
 
-## Contrib
+# How to Contribute
```

Comment:
In addition to this we should have them take GH issues for new feature requests. If we have one example GH issue that will be nice and we should link the appropriate GH label (SAMPLE_REQUEST) link here.

Reply:
Added